src/experiments.js

```diff
@@ -37,6 +42,9 @@ const COOKIE_EXPIRATION_INTERVAL = COOKIE_MAX_AGE_DAYS * 24 * 60 * 60 * 1000;
 /** @type {Object<string, boolean>} */
 let toggles_ = null;
 
+/** @type {!Promise} */
+const originTrialsPromise = enableExperimentsForOriginTrials(self);
```

Comment:
You can create a new Promise, store the resolve function and call it manually when it completes. 

At 5ms it might be fine to have the entire thing in a single chunk.